environment/simulation/utils/simulation_pymunk_utils.py

```diff
@@ -39,7 +39,7 @@ def create_bot(position_x, color, screen_size, max_distance_from_threshold=100):
     radius = BOTS_RADIUS
     mass = BOTS_MASS
     body = pymunk.Body(mass, moment=pymunk.moment_for_circle(mass, inner_radius=0, outer_radius=radius))
-    body.position = (random.randint(min_pos, max_pos), screen_size[1])
+    body.position = (random.randint(min_pos, max_pos), 20)
```

Comment:
Czemu akurat 20?

Reply:
Kuleczki wtedy sie jakos ladniej inicjują kiedy spadają przez ułamek sekundy z niziutkiej wysokości a nie spawnują prosto na mapie bo wtedy czasem dziwnie się zachowują. Plus moge docommitować tak:
```suggestion
    safety_y_offset = 20
    body.position = (random.randint(min_pos, max_pos), safety_y_offset)
```

Reply:
O, super :3